pages/task/content/status.js

```diff
@@ -95,7 +100,8 @@ module.exports = {
   },
   'with-ntco': {
     state: 'Awaiting endorsement',
-    log: 'Submitted by'
+    log: 'Submitted by',
+    currentlyWith: 'Currently with: Named Training and Competency Officer'
```

Comment:
Is this status _actually_ used for things that are with NTCO, or is it only there for b/c?

Reply:
think it's legacy.